### PR TITLE
Features/models

### PR DESCRIPTION
**Pull request by Chester97**

### CODE REVIEW

File: src/models/Users.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { observable, action, computed } from "mobx";
+
+export default class Users {
```
Comment (kamil7x) [inline src/models/Users.js]: `User`. Model is describing one item.

Comment (kamil7x) [reply src/models/Users.js]: Same for file names

**Comment by kamil7x** [comment]
`feature/models`, not `featureS/models`. Remember that :)

File: src/index.js [hunk 2]
```diff
@@ -4,5 +4,8 @@ import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
+
```
Comment (kamil7x) [inline src/index.js:7]: What is that?

File: src/models/Users.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        devices.push(device)
+      }
+      removeDevice(deviceId) {
+
+      }
+
+      @observable places = [];
+
+      constructor({id,avatar,firstName,lastName,email,login,officeId,isAdmin}) {
+          this.id=id;
```
Comment (kamil7x) [inline src/models/Users.js]: It's better practice to use spaces for readability.
```this.id = id;```
```{ id, avatar }```
etc

I think ESlint should check this, not sure why it didn't work this time

**Comment by Chester97** [comment]
OK, boss!

File: src/App.js [hunk 4]
```diff
@@ -1,19 +1,28 @@
 import React, { Component } from 'react';
+import {observer} from "mobx-react";
 import logo from './logo.svg';
 import './App.css';
 
+import Users from './models/Users';
```
Comment (kamil7x) [inline src/App.js]: Try to not import things, that you are not going to use in this file.
I really think, we need to check ESLint configuration :D

**Review by kamil7x — CHANGES_REQUESTED** [review]

File: src/models/User.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { observable, action, computed } from "mobx";
 import { remove } from 'lodash';
 
-export default class Users {
+export default class Userf {
```
Comment (kamil7x) [inline src/models/User.js]: Userf? :D

**Review by kamil7x — APPROVED** [review]

**Pull request merged**